pinot-controller/src/main/java/org/apache/pinot/controller/recommender/io/InputManager.java

```diff
@@ -169,6 +172,33 @@ private void regulateCardinalityForAll() {
     });
   }
 
```

Comment:
Let's delete regulateCardinalityForAll() method. Should no longer be used after all the cardinality related changes in this PR.

Reply:
Done.